frontend/talentsearch/src/js/components/workPreferencesForm/WorkPreferencesForm.tsx

```diff
@@ -39,6 +39,24 @@ export interface WorkPreferencesFormProps {
   ) => Promise<UpdateWorkPreferencesMutation["updateUserAsUser"]>;
 }
 
+interface WithEllipsisPrefixProps {
+  children: React.ReactNode;
+}
+const WithEllipsisPrefix = ({ children }: WithEllipsisPrefixProps) => {
+  const { formatMessage } = useIntl();
+  const ellipsisPrefix = formatMessage({
+    defaultMessage: "...",
```

Comment:
I'm not sure if it would work but could we use a `&hellip;` here or would it just render as text rather than `...`?

Reply:
I can scope that! I tried to get non-breaking space working via that syntax, and it printed out, but maybe that's a specialcase for whitespace chars 🤷

Reply:
Hm, seems that only unicode works, not html entities.

```diff
diff --git a/frontend/talentsearch/src/js/lang/fr.json b/frontend/talentsearch/src/js/lang/fr.json
index fe7a5063f..34928125a 100644
--- a/frontend/talentsearch/src/js/lang/fr.json
+++ b/frontend/talentsearch/src/js/lang/fr.json
@@ -1,6 +1,6 @@
 {
   ".ellipsis": {
-    "defaultMessage": "… "
+    "defaultMessage": "\u2026 "
   },
   "+ZXZj+": {
     "defaultMessage": "S’il n’y a aucun candidat qui répond à vos critères <a>Communiquez avec nous! </a>",
```

Any pref? My loose one is to stick with "..." (3 chars). The only reason I used the single char for french was because it was used in a bunch of translations, and I thought maybe it was a french language preference. But I suspect it was just the word processor auto-correcting.

I suppose I could ask @gobyrne: is the single char "horizontal ellipsis" preferred for french translations?

If not, I'm inclined to just standardize on "3 dot chars". Better to be consistent, bc they look slightly diff. And 3 dots is just easier to type.

Reply:
✅  Reverted to 3-char version for French